src/sidebar/services/test/tags-test.js

```diff
@@ -56,6 +56,12 @@ describe('TagsService', () => {
         count: 1,
         updated: stamp,
       },
+      'žŸ¡¢£¤¥¦§': {
+        // ASCII chars
```

Comment:
All the other test cases are technically ASCII as well. Something like "Non-word ASCII chars" would be better.